db/migrate/20241205222413_add_depositors_and_managers_join_tables.rb

```diff
@@ -0,0 +1,15 @@
+class AddDepositorsAndManagersJoinTables < ActiveRecord::Migration[8.0]
+  def change
+    create_join_table :collections, :users, table_name: :depositors do |t|
+      t.index [:collection_id, :user_id], unique: true
+    end
+
+    create_join_table :collections, :users, table_name: :managers do |t|
+      t.index [:collection_id, :user_id], unique: true
+    end
+
+    create_join_table :collections, :users, table_name: :reviewers do |t|
```

Comment:
Just a note that I felt like it made sense to make the reviewers join table now so that it's in place when we add the review workflow/reviewers, but I can remove if we don't want to wait on including this.